src/components/styles/ProjectPreview.styled.ts

```diff
@@ -0,0 +1,61 @@
+import styled from 'styled-components'
+
+export const PreviewText = styled.div<{ hasAnimation: boolean }>`
+  font-size: 0.9rem;
+  color: ${(props) => props.theme.palette.common.white};
+  @keyframes fadeIn {
+    0% {
+      opacity: 0;
+    }
+    100% {
+      opacity: 1;
+    }
+  }
+  ${(props) => {
+    if (props.hasAnimation) {
+      return `
+      animation-name: fadeIn;
+      animation-delay: 1s;
+      animation-duration: 1.5s;
+      animation-fill-mode: both; 
+      `
+    }
+    return `
+      animation-name: none;
+    `
+  }}
+`
+export const PreviewTitle = styled.div<{ fontSize?: string }>`
+  width: fit-content;
+  justify-content: center;
+  align-items: center;
+  color: ${(props) => props.theme.palette.common.white};
+  font-size: ${(props) => props.fontSize};
+  font-weight: 600;
+  border: 0;
+  background-color: ${(props) => props.theme.palette.common.black};
+  transition: background-image 0.4s ease-in-out;
+  &:hover {
+    cursor: pointer;
```

Comment:
could this work with this outside hover?

Reply:
c94d62e8b8e3da05f86fb0a66eac078a297ff9c3